File: benchmark/sum.js
const start = Date.now();
let sum = 0;
for(let i=0;i<10000000;i++){
    sum += i;
}
const duration = (Date.now() - start) / 1000;
console.log(sum);
console.log(`time: ${duration.toFixed(3)}`);

File: benchmark/sum.py
import time

def main():
    start = time.perf_counter()
    s = 0
    for i in range(10_000_000):
        s += i
    duration = time.perf_counter() - start
    print(s)
    print(f"time: {duration:.3f}")

if __name__ == "__main__":
    main()
